docs/getting-started/dev-portal/web-component.md

```diff
@@ -59,7 +59,7 @@ Alternatively, copy and paste this into an `index.html` and open it in a browser
 </html>
 ```
 
-5. Finally, let's serve it with a HTTP server. Grab any HTTP server you like, maybe the NPM [http-server](https://www.npmjs.com/package/http-server).
+5. Finally, let's serve it with a HTTP server. Grab any HTTP server you like, such as the NPM [http-server](https://www.npmjs.com/package/http-server).
```

Comment:
I'm curious if it's more grammatically correct to use 'an' before HTTP since most pronounce it as 'aych'? 
```suggestion
5. Finally, let's serve it with an HTTP server. Grab any HTTP server you like, such as the NPM [http-server](https://www.npmjs.com/package/http-server).
```

Reply:
I had to look it up but I think `an` is correct! https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html